wpa_supplicant/p2p_supplicant.c

```diff
@@ -5210,7 +5210,7 @@ static int wpas_p2p_peer_go(struct wpa_supplicant *wpa_s,
 	updated = os_reltime_before(&wpa_s->p2p_auto_started,
 				    &bss->last_update);
 	wpa_printf(MSG_DEBUG, "P2P: Current BSS entry for peer updated at "
-		   "%lld.%06lld (%supdated in last scan)",
+		   "%ld.%06ld (%supdated in last scan)",
```

Comment:
is this format architecture dependent? I'm guessing this is a warning if you do it wrong?

Reply:
This is actually reverting the changes I did when porting hostap to upstream zephyr, so I added the `%lld`, but then dbe9b099edf2c54ce2c04c93ac6b952dbb798fe9 changed the `os_time_t` back to the original (in order to not do so much changes in the code). But some of my conversions were missed.